action.yml

```diff
@@ -36,6 +36,15 @@ inputs:
     description: "The name to use for the SBOM file generated by this action"
     required: false
 
+  syft-version:
+    required: false
+    description: "Syft version string to use"
```

Comment:
Could we make this description a little more clear? (How is this being used? What's the difference between a Syft version and a Syft version string? Is this used to specify which version of Syft will get executed?)